sencha-workspace/packages/slate-cbl/src/view/tasks/StudentTaskForm.js

```diff
@@ -575,7 +579,7 @@ Ext.define('Slate.cbl.view.tasks.StudentTaskForm', function() {
                         me.getExpirationDateOverrideField()
                     ]
                 },
-                me.getTaskAttachmentsField(),
+                taskAttachmentsField,
```

Comment:
leave this as it was, and instead implement an `updateDisplayRemovedTasks(displayRemovedTasks)` method that calls `me.getTaskAttachmentsField().setDisplayRemoved(displayRemovedTasks)`—passing down the config value through the slotted component instance independent of render lifecycle state